src/utils/states.js

```diff
@@ -2,7 +2,6 @@ export const STATE_COLORS = {
   Failed: '#eb0000',
   Pending: '#b9dcff',
   Scheduled: '#ffbe1e',
-  ScheduledAlt: '#ffeec4',
```

Comment:
It's ok if we have this here since it's for styling and isn't displayed in any list - can probably revert this line and the changes to `vuetify.js` (unless there's a different complication here i'm not aware of)